src/com/oltpbenchmark/benchmarks/auctionmark/procedures/CloseAuctions.java

```diff
@@ -193,26 +206,28 @@
 		};
 		output_rows.add(row);
 
-                sb = new StringBuilder();
-                sb.append("SELECT i_end_date");
-                sb.append(" FROM ");
-                sb.append(AuctionMarkConstants.TABLENAME_ITEM);
-                sb.append(" WHERE i_id = ");
-                sb.append(itemId);
-		sb.append( " AND i_end_date < '" );
-		sb.append( currentTime );
-		sb.append( "'" );
+		/*
+			// Retrieve the feedback for the buying user from the last 30 days
+			sb = new StringBuilder();
+			sb.append( "SELECT AVG(uf_rating) AS av FROM " );
+			sb.append( AuctionMarkConstants.TABLENAME_USERACCT_FEEDBACK );
+			sb.append( " WHERE uf_u_id = " );
+			sb.append( sellerId );
+			sb.append( " AND uf_date > '" );
+			Timestamp thirtyDays = new Timestamp(currentTime.getTime() - 2592000000L);
+			sb.append( thirtyDays );
+			sb.append( "'" );
```

Comment:
Get the average feedback for the selling user of this auction within the last 30 days. This is the "extra query" that highlights the performance difference between Apollo and Scalpel.